src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetAccountPAYESchemes/WhenIGetAccountPayeSchemes.cs

```diff
@@ -13,7 +14,7 @@
 
 namespace SFA.DAS.EmployerAccounts.UnitTests.Queries.GetAccountPAYESchemes
 {
-    class WhenIGetAccountPayeSchemes : QueryBaseTest<GetAccountPayeSchemesQueryHandler, GetAccountPayeSchemesQuery, GetAccountPayeSchemesResponse>
+    class WhenIGetAccountPayeSchemes : QueryBaseTest<GetAccountPayeSchemesForAuthorisedUserQueryHandler, GetAccountPayeSchemesForAuthorisedUserQuery, GetAccountPayeSchemesResponse>
```

Comment:
Tests should be specific to the class under test rather than testing 2 at once.

Reply:
As per discussion; common code extracted into new service `PayeSchemesService`

Query handlers both updated to call this service.

No update of test styles or scenarios, but scenarios covering expected results for original command handler now applied to service.